tests/test_integration.py

```diff
@@ -455,6 +456,21 @@ def test_info(self):
         self.vm.shutdown()
         host_info(VM_HOSTNAME)
 
+    @patch('igvm.vm.VM.vm_performance_value', return_value=5.0)
+    def test_igvm_migration_log(self, mock_vm_performance_value):
+        regex = r'\A([0-9]{10}\s[-|+][0-9]{1,3})+\Z'
+
+        for hv in self.hvs:
+            if hv.dataset_obj['hostname'] == self.vm.dataset_obj['hypervisor']:
+                hv.dataset_obj['igvm_migration_log'] = ''
+                hv.dataset_obj.commit()
+                self.assertEqual(str(hv.dataset_obj['igvm_migration_log']), '')
+
+                hv.hv_add_migration_log(self.vm, '-')
+                hv.hv_add_migration_log(self.vm, '+')
+                for entry in hv.dataset_obj['igvm_migration_log']:
+                    self.assertRegex(entry, regex)
```

Comment:
This should be already enforced by Serveradmin but it does not harm I guess.

Reply:
i think we should even check for the exact expected migration log entry, not for any regex

Reply:
Good point.

Reply:
I reworked it, checking the log on the source hypervisor, migrating the VM and checking the log on the destination hypervisor then.